scipy/spatial/tests/test_distance.py

```diff
@@ -317,6 +317,22 @@ def test_cdist_dtype_equivalence(self):
                     for new_type in test[1]:
                         y2 = cdist(new_type(X1), new_type(X2), metric=metric)
                         _assert_within_tol(y1, y2, eps, verbose > 2)
+                        
+    def test_cdist_out(self):
+        esp = 1e-07
+        out1 = np.empty((10, 20), dtype=np.double)
```

Comment:
To improve readability, what about change the order so to group together the different aspects you are testing? You may also add a line of comment saying what are you testing  (E.g. C-contiguous, wrong shape..)